docs/wxm-devices/helium-bundle/helium-install-weather-station.mdx

```diff
@@ -10,7 +10,7 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 ## Prerequisites for proper installation
 1. Pole of at least 2m/6.5ft (not included)
 1. Base or bolts to secure in place (not included)
-1. Location and spacing (<a href="/deployment-examples">Best practices for proper station installation</a>)
+1. Location and spacing (<a href="https://docs.weatherxm.com/wxm-devices/helium-bundle/helium-install-weather-station#best-practices-for-proper-station-installation">Best practices for proper station installation</a>)
```

Comment:
Hmm.. It opens a new tab, let's do it relative, so replace this URL with:
`/wxm-devices/helium-bundle/helium-install-weather-station#best-practices-for-proper-station-installation`

I think that would work.